nhrpd/nhrp_vty.c

```diff
@@ -1023,9 +1023,7 @@ DEFUN(show_ip_nhrp, show_ip_nhrp_cmd,
 	} else if (argv[3]->text[0] == 's') {
 		nhrp_shortcut_foreach(ctx.afi, show_ip_nhrp_shortcut, &ctx);
 	} else {
-		if (!ctx.json)
-			vty_out(vty, "Status: ok\n\n");
```

Comment:
So, this is nonsense at all (writing this status as OK) if I understand it correctly?

Reply:
Yes. It is nonsense to always write it as OK.